linx-impulse/actions/analytics/sendEvent.ts

```diff
@@ -0,0 +1,261 @@
+import { AppContext } from "../../mod.ts";
```

Comment:
instead of anonymous types

type X = {}

use interface X {}

So we have better names being generated on admin